npm-shrinkwrap.json

```diff
@@ -2124,7 +2139,7 @@
         "xtend": {
           "version": "4.0.1",
           "from": "xtend@>=4.0.0 <5.0.0",
-          "resolved": "https://registry.npmjs.org/xtend/-/xtend-4.0.1.tgz",
+          "resolved": "https://lunabuild.akamai.com/nexus/content/groups/npm-all/xtend/-/xtend-4.0.1.tgz",
```

Comment:
seems like this might be related to : https://github.com/strongloop/fsevents/issues/141. they just updated fsevents an hour ago, i bet their latest version has this same issue.